tests/actions/PolicyTest.js

```diff
@@ -0,0 +1,189 @@
+            expect(policyMembers[ESH_ACCOUNT_ID].role).toBe(CONST.POLICY.ROLE.ADMIN);
+
+            let allReports = await new Promise((resolve) => {
+                const connectionID = Onyx.connect({
+                    key: ONYXKEYS.COLLECTION.REPORT,
+                    waitForCollectionCallback: true,
+                    callback: (reports) => {
+                        Onyx.disconnect(connectionID);
+                        resolve(reports);
+                    },
+                });
+            });
+
+            // Three reports should be created: #announce, #admins and expense report
+            const workspaceReports = _.filter(allReports, (report) => report.policyID === policyID);
+            expect(_.size(workspaceReports)).toBe(3);
+            _.forEach(workspaceReports, (report) => {
+                expect(report.pendingFields.addWorkspaceRoom).toBe(CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
+                expect(report.participantAccountIDs).toEqual([ESH_ACCOUNT_ID]);
+                switch (report.chatType) {
+                    case CONST.REPORT.CHAT_TYPE.POLICY_ADMINS: {
+                        adminReportID = report.reportID;
+                        break;
+                    }
+                    case CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE: {
+                        announceReportID = report.reportID;
+                        break;
+                    }
+                    case CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT: {
+                        expenseReportID = report.reportID;
+                        break;
+                    }
+                    default:
+                        break;
+                }
+            });
+
+            let reportActions = await new Promise((resolve) => {
+                const connectionID = Onyx.connect({
+                    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+                    waitForCollectionCallback: true,
+                    callback: (actions) => {
+                        Onyx.disconnect(connectionID);
+                        resolve(actions);
+                    },
+                });
+            });
+
+            // Each of the three reports should have a a `CREATED` action.
+            let adminReportActions = _.values(reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${adminReportID}`]);
+            let announceReportActions = _.values(reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${announceReportID}`]);
+            let expenseReportActions = _.values(reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${expenseReportID}`]);
+            let workspaceReportActions = _.concat(adminReportActions, announceReportActions, expenseReportActions);
+            _.forEach([adminReportActions, announceReportActions, expenseReportActions], (actions) => {
+                expect(_.size(actions)).toBe(1);
+            });
+            _.forEach(workspaceReportActions, (reportAction) => {
```

Comment:
NAB, remove this single-use variable:

```suggestion
            _.forEach([...adminReportActions, ...announceReportActions, ...expenseReportActions], (reportAction) => {
```